decode_code.rb

```diff
@@ -0,0 +1,29 @@
+# Decode Letter From Morse Code
+
+def decode_char(char)
+  morse_alphabet = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E',
+    '..-.' => 'F', '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J',
+    '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O',
+    '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T',
+    '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y',
+    '--..' => 'Z'
+  }
+  morse_alphabet[char]
+end
+
+puts decode_char('--')
+# Decode a Word Using decode_char function
+def decode_word(word)
+  decoded_chars = word.split.map { |char| decode_char(char) }
+  decoded_chars.join(' ')
+end
+
+puts decode_word('-- -.--   -. .- -- .')
+
+# Decode a Sentence Using decode_word function
+def decode_sentence(sentence)
+  decoded_words = sentence.split.map { |word| decode_word(word) }
+  decoded_words.join(' ')
+end
```

Comment:
- [ ] I recommend changing the code to distinguish the words between the spaces, and pay attention to the spaces on the join method, in word there aren't spaces, but there are in sentences. 👇🏽 

```ruby
def decode_word(word)
  decoded_chars = word.split.map { |char| decode_char(char) }
  decoded_chars.join('')
end

puts decode_word('-- -.--   -. .- -- .')

# Decode a Sentence Using decode_word function
def decode_sentence(sentence)
  decoded_words = sentence.split('   ').map { |word| decode_word(word) }
  decoded_words.join(' ')
end
puts decode_sentence('.-  -... --- -..-   ..-. ..- .-.. .-..   --- ..-.  .-. ..- -... .. . ...')
```

Reply:
Hi @franclobo . I am not sure I understand what you mean. Can you please clarify your comment and give some example?
Thanks.